DEPS

```diff
@@ -12,7 +12,7 @@ vars = {
   'chromium_version':
     '76.0.3809.23',
   'node_version':
-    'dee0db9864a001ffc16440f725f4952a1a417069',
+    '229bd3245b2f54c12ea9ad0abcadbc209f8023dc',
```

Comment:
there was a conflict here, which version should be used?

Reply:
229bd3245b2f54c12ea9ad0abcadbc209f8023dc is the right commit.